### ISSUE TITLE
Search ignores queries pasted with a mouse

### ISSUE THREAD

**Issue by InfernalLV** [odoo/odoo#16285]
Impacted versions: Enterprise v10

Steps to reproduce:
- copy any text
- paste *via a mouses context menu (rightclick -> paste)* in a any search view
- press enter

Current behavior: the search term disappears
Expected behavior: filter the records

The paste event does not prompt the user which field/filter to query

**Comment by Yenthe666** [comment]
Hi @InfernalLV,

Thanks for your report.
This is indeed the case. @nim-odoo perhaps this is something for you?

Regards,
Yenthe

**Comment by xmo-odoo** [comment]
It looks like the mouse-paste event doesn't properly trigger whatever event the FAYT is hooked to (input? onchange?) and thus doesn't open the completion dropdown. Pasting with the keyboard however looks properly.

**Comment by Yenthe666** [comment]
Indeed. This behaviour is still relevant at this point though. So, do we consider it something to fix or as desired behaviour? (CC @ged-odoo as it is JS related really).

**Comment by ged-odoo** [comment]
looks like a bug, low priority i would say. i suppose that the search view should also listen to the 'change' events, but we need to avoid doing 2 searches... I think that, unless a PR is submitted, it has a low chance of being fixed in 10.0. But we may do that in 11.0, or more likely in master, when we rework the controlpanel/search view

**Comment by Yenthe666** [comment]
Alright! Let us keep it wishlist for now then.

**Comment by C3POdoo** [comment]
Dear @karlis-dreizis,

Thank you for your report but we are closing it due to inactivity.
We apology if we could not look at your request in time.
If your report still makes sense, don't hesitate to reopen a new one. We will try to check it as soon as possible.

_This is an automated message._
